### PR TITLE
[PWD-2579] Adjustment - Custom Training

### PR DESCRIPTION
**Pull request by haikal1223**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

File: src/views/custom-training/components/StickyNavbar.tsx
Comment (hiljaker) [inline src/views/custom-training/components/StickyNavbar.tsx:1]: nih belum dihapus

File: src/pages/CorporateTraining.tsx [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+
+  if (type === 'fixed-programs') {
+    return <CorporateTrainingPage />
+  }
+
+  if (type === 'custom-training') {
+    return <CustomTrainingView />
+  }
+
+  return <NotFound />
```
Comment (shadiqfauzi) [inline src/pages/CorporateTraining.tsx:17]: redirect programatically ke not found aja possible ga? jadi ga perlu import component not found disini

File: src/views/custom-training/components/StickyNavbar.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Box, Typography } from '@mui/material'
```
Comment (shadiqfauzi) [inline src/views/custom-training/components/StickyNavbar.tsx:1]: ini component apa?

Comment (haikal1223) [reply src/views/custom-training/components/StickyNavbar.tsx:1]: ini keliwat a wkwk. Gw apus

File: src/pages/CorporateTraining.tsx (hunk 1 above)
Comment (haikal1223) [reply src/pages/CorporateTraining.tsx:17]: wokehh

File: src/views/custom-training/components/StickyNavbar.tsx
Comment (haikal1223) [reply src/views/custom-training/components/StickyNavbar.tsx:1]: 🗿

**Review by shadiqfauzi — APPROVED** [review]

**Review by Vesals — APPROVED** [review]

**Review by gangsararyop — APPROVED** [review]

**Pull request merged**